### DISCUSSION TITLE
Web browser?

### DISCUSSION THREAD

**Discussion by dreams897** [OnionUI/Onion#1364] [Ideas]
How hard would it be to implement a web browser or is there already such an implementation that I am unaware of?

**Comment by Starbrush** [comment]
theoretically it would run, but it takes a lot of work to adapt the browser to run and install files, as it consumes a lot of memory, but it would be very good and a spectacular addition

**Comment by edmunek** [comment]
somehow, based on years of IT experience and type of a geek person, I would be against this idea.

**Reply by Starbrush** [reply]
I have a different view, but I respect your opinion, I know it's difficult, and it might even get weird, but the majority of the community would like this addition, there are several situations where this would be important, like going to someone else's house, traveling, and among others.

**Reply by dariu76** [reply]
download roms without going through a PC...

**Reply by Starbrush** [reply]
Yep, I had forgotten to mention: what could we do with a web browser? well we could download any type of file, movies, roms, manuals, and even apps it would be very good and save a lot of time

**Comment by Nighted** [comment]
I think it would be superfluous. What next, Photoshop?

**Reply by Starbrush** [reply]
It's complicated, I think just downloading files is more than enough, I'm not going to turn on the miyoo to watch a video on YouTube, it would be something very simple to navigate, but of course it depends on how hard it is to make a web browser for the miyoo, but in comparison the Nintendo ds lite had a web browser, so it's kinda possible

**Reply by Nighted** [reply]
It's not complicated. That's the point.
